From 7248d38028c0939c74865daeab66fa7e36f5bedf Mon Sep 17 00:00:00 2001
From: LuK1337 <priv.luk@gmail.com>
Date: Sat, 31 Mar 2018 19:05:12 +0200
Subject: [PATCH 64/74] fm_hci/helium: Exclude from OSS builds

* These libraries depend on closed source
  headers which we don't include in OSS builds.

Change-Id: I8fc82ff468e1f58eebd4fe0ab4b3f832cd7e2bab
---
 fm_hci/Android.mk | 2 ++
 helium/Android.mk | 2 ++
 2 files changed, 4 insertions(+)

diff --git a/fm_hci/Android.mk b/fm_hci/Android.mk
index bd7340b..64b6cd5 100644
--- a/fm_hci/Android.mk
+++ b/fm_hci/Android.mk
@@ -1,3 +1,4 @@
+ifneq ($(QCPATH),)
 LOCAL_PATH := $(call my-dir)
 
 include $(CLEAR_VARS)
@@ -27,3 +28,4 @@ LOCAL_MODULE_TAGS := optional
 LOCAL_MODULE_CLASS := SHARED_LIBRARIES
 
 include $(BUILD_SHARED_LIBRARY)
+endif
diff --git a/helium/Android.mk b/helium/Android.mk
index 91c297c..f7fc1d7 100644
--- a/helium/Android.mk
+++ b/helium/Android.mk
@@ -1,3 +1,4 @@
+ifneq ($(QCPATH),)
 LOCAL_PATH := $(call my-dir)
 include $(CLEAR_VARS)
 
@@ -20,3 +21,4 @@ LOCAL_MODULE := fm_helium
 LOCAL_MODULE_TAGS := optional
 
 include $(BUILD_SHARED_LIBRARY)
+endif
-- 
2.17.1

